RafRate.API/Controllers/AuthenticationController.cs

```diff
@@ -8,10 +11,32 @@ namespace RafRate.API.Controllers
     public class AuthenticationController : ControllerBase
```

Comment:
You should rename `ControllerBase` to `BaseController`

Reply:
![image](https://user-images.githubusercontent.com/70603829/184544904-5bcac9c0-bea3-40be-a4cb-054df97c916c.png)

Reply:
![image](https://user-images.githubusercontent.com/36763052/184547457-389ba6b7-ea1d-4c91-89a9-8811779ee592.png)